### DISCUSSION TITLE
Github student pack Can't receive email verification for student mail

### DISCUSSION THREAD

**Discussion by birajcodes** [community/community#108934] [GitHub Education]
### Select Topic Area

General

### Body

Hello I am trying to access github student developer pack for learning , my college have provided me  this email and i am trying to add it on my add mail and it says send but i am not receiving it in my university mail inbox i am been trying from past week , i dont know why i am not receiving please help ,

**Comment by mesupreneur** [comment]
It sounds like you're experiencing difficulty receiving emails sent to your university email address  from GitHub. Contact University IT Support: Reach out to your university's IT support or help desk. They may be able to assist you in troubleshooting any issues with receiving external emails or identify any specific settings that need to be adjusted.

Verify Email Address: Double-check that the email address you provided to GitHub is correct. If there's a typo or mistake in the email address, you won't receive the emails.

Alternative Email: If possible, consider using an alternative email address (such as a personal email) to register for the GitHub Student Developer Pack. This can help bypass any issues with your university email account.

GitHub Support: If the issue persists, you can also reach out to GitHub support for assistance. They may be able to provide further guidance or troubleshoot any issues from their end.

**Reply by queenofcorgis** [reply]
@mesupreneur 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @birajcodes ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

I've removed your private information from this post and comments as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
